bench.py

```diff
@@ -197,6 +201,8 @@ def parse_args(parser, size=None, loop_types=(),
         except ImportError:
             print('Failed to import daal4py.sklearn.patch_sklearn.'
                   'Use stock version scikit-learn', file=sys.stderr)
+    else:
+        params.device = None
```

Comment:
I think we should check if the device parameter is passed by the user and print a warning that it is useless in that case - for clarity